cluster/src/main/scala/io/snappydata/metrics/SnappyMetricsSystem.scala

```diff
@@ -85,7 +81,6 @@ object SnappyMetricsSystem {
   }
 
   def putMembersDiskStoreIdInRegion(): Unit = {
-
     val membersBuff = SnappyTableStatsProviderService.getService.getMembersStatsFromService
     for ((k, v) <- membersBuff) {
       val shortDirName = v.getUserDir.substring(
```

Comment:
Better to move `System.getProperty("file.separator")` outside the for loop. Also, to take a reference of `Misc.getMemStore.getMetadataCmdRgn` outside the for loop.

Reply:
Added suggested code changes